src/SFA.DAS.EAS.Web/Controllers/EmployerManageApprenticesController.cs

```diff
@@ -203,18 +199,72 @@ public async Task<ActionResult> WhenToApplyChange(string hashedAccountId, string
                 return View(new OrchestratorResponse<WhenToMakeChangeViewModel>() { Data = viewResponse.Data });
             }
 
-            return RedirectToRoute("StatusChangeConfirmation", new { whenToMakeChange = model.WhenToMakeChange, dateOfChange = response.DateOfChange });
+            return RedirectToRoute("MadeRedundant", new { whenToMakeChange = model.WhenToMakeChange, dateOfChange = response.DateOfChange });
+        }
+
+        [HttpGet]
+        [Route("{hashedApprenticeshipId}/details/statuschange/{changeType}/maderedundant", Name = "MadeRedundant")]
+        [OutputCache(CacheProfile = "NoCache")]
+        public async Task<ActionResult> HasApprenticeBeenMadeRedundant(string hashedAccountId, string hashedApprenticeshipId, ChangeStatusType changeType, DateTime? dateOfChange, WhenToMakeChangeOptions whenToMakeChange)
+        {
+            if (!await IsUserRoleAuthorized(hashedAccountId, Role.Owner, Role.Transactor))
+                return View("AccessDenied");
+
+            var response = await _orchestrator.MakeApprenticeRedundant(hashedAccountId, hashedApprenticeshipId, changeType, dateOfChange, whenToMakeChange, OwinWrapper.GetClaimValue(@"sub"), null);
+
+            return View(new OrchestratorResponse<RedundantApprenticeViewModel>

```

Comment:
Can we just do View(response)?

Reply:
i was just following the pattern used in the file.